app/components/ListItemSwitch/ListItemSwitch.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { View, ViewStyle, StyleProp } from "react-native";
+import { Switch } from "react-native-gesture-handler";
+
+import styles from "./ListItemSwitch.styles";
+import Text from "../Text";
+
+interface Props {
+  title: string;
+  value: boolean;
+  onChange: () => void;
+  showTopLine?: boolean;
+  showBottomLine?: boolean;
+}
+
+const ListItemSwitch: React.FC<Props> = ({ title, value, onChange, showTopLine, showBottomLine }) => {
+  const containerStyle: StyleProp<ViewStyle> = [styles.container];
+  const switchStyle: StyleProp<ViewStyle> = styles.switch;
+
+  if (showTopLine) {
+    containerStyle.push(styles.topLine);
+  }
+
+  if (showBottomLine) {
+    containerStyle.push(styles.bottomLine);
+  }
+
+  return (
+    <View style={containerStyle}>
+      <Text.Secondary numberOfLines={1}>{title}</Text.Secondary>
+      <Switch
```

Comment:
The switch needs to be center
<img width="421" alt="Screenshot 2023-02-03 at 10 33 12" src="https://user-images.githubusercontent.com/8709134/216575887-1f0a8f35-7832-4939-a0b2-550f42c69aae.png">

Reply:
I think I'll need some help with this one. I added some padding to the <Switch> descendant component of <ListItemSwitch> in the most recent push to maybe help fix this, but I can't actually perform a visual check since I don't own anything that can run or emulate iOS (as far as I know).

It seems to be working in Android for me.
<img width="224" alt="units_toggle_center_android" src="https://user-images.githubusercontent.com/32628434/218643406-a9dc1b1e-c463-49fb-a86f-705c078d79ff.png">